MigrateServicePreferenceFromLegacy/handler.ts

```diff
@@ -12,6 +12,8 @@ import {
 } from "@pagopa/io-functions-commons/dist/src/utils/cosmosdb_model";
 import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
 import * as a from "fp-ts/lib/Array";
+import * as e from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
 import * as o from "fp-ts/lib/Option";
 import * as te from "fp-ts/lib/TaskEither";
 import * as t from "io-ts";
```

Comment:
I'd rather maintain the same convention about naming imports from fp-ts. Can we change to upper case names?

Reply:
Yes sure, I missed something that was already defined but I''ll fix it.

Reply:
Done [here](https://github.com/pagopa/io-functions-app/pull/202/commits/9745f3ef828fcb0c9abfe7074a5311e872f410e7)